<launch>
  <node pkg="riptide_controllers" type="command_combinator" name="command_combinator" output="screen" >
    <rosparam file="$(find riptide_controllers)/cfg/max_accel.yaml" command="load" />
  </node>
</launch>
